bitnami/memcached/values.yaml

```diff
@@ -227,82 +267,261 @@ nodeAffinityPreset:
+  clusterIP: ""
+  ## @param service.loadBalancerIP Memcached service Load Balancer IP
+  ## ref: https://kubernetes.io/docs/user-guide/services/#type-loadbalancer
+  ##
+  loadBalancerIP: ""
+  ## @param service.loadBalancerSourceRanges Memcached service Load Balancer sources
+  ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  ## e.g:
+  ## loadBalancerSourceRanges:
+  ##   - 10.10.10.0/24
+  ##
+  loadBalancerSourceRanges: []
+  ## @param service.externalTrafficPolicy Memcached service external traffic policy
+  ## ref https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+  ##
+  externalTrafficPolicy: Cluster
+  ## @param service.annotations Additional custom annotations for Memcached service
+  ##
+  annotations: {}
+  ## @param service.extraPorts Extra ports to expose in the Memcached service (normally used with the `sidecar` value)
+  ##
+  extraPorts: []
+
+## @section Other Parameters
+
+## Service account for Memcached to use.
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
 ##
 serviceAccount:
-  ## @param serviceAccount.create Enable creation of ServiceAccount for memcached pods
+  ## @param serviceAccount.create Enable creation of ServiceAccount for Memcached pod
   ##
-  create: true
+  create: false
   ## @param serviceAccount.name The name of the ServiceAccount to use.
-  ## If not set and create is true, a name is generated using the memcached.serviceAccountName template
+  ## If not set and create is true, a name is generated using the common.names.fullname template
   ##
   name: ""
-  ## @param serviceAccount.automountServiceAccountToken Enable/disable auto mounting of the service account token
+  ## @param serviceAccount.automountServiceAccountToken Allows auto mount of ServiceAccountToken on the serviceAccount created
+  ## Can be set to false if pods using this serviceAccount do not need to use K8s API
   ##
   automountServiceAccountToken: true
-## Persistence - used for dumping and restoring states between recreations
-## Ref: https://github.com/memcached/memcached/wiki/WarmRestart
+  ## @param serviceAccount.annotations Additional custom annotations for the ServiceAccount
+  ##
+  annotations: {}
+
+## @section Persistence parameters
+
+## Enable persistence using Persistent Volume Claims
+## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
 ##
 persistence:
-  ## @param persistence.enabled Enable persistence using PVC (Requires architecture: "high-availability")
+  ## @param persistence.enabled Enable Memcached data persistence using PVC. If false, use emptyDir
   ##
   enabled: false
-  ## @param persistence.storageClass PVC Storage Class for Memcached volume
+  ## @param persistence.existingClaim Name of an existing PVC to use (only when deploying a single replica)
+  ##
+  existingClaim: ""
```

Comment:
`existingClaim` defined but not used.

Reply:
I'll remove it since this parameter since it's only possible to enable persistence when using a statefulset and you'll need N claims in this case.

Reply:
That works for me 👍